docs/wiki/framework/attach-framework.md

```diff
@@ -0,0 +1,94 @@
+---
+layout: wiki
+title: Attach Framework
+description: Explains how to add items to ACE's Attach system.
+group: framework
+order: 0
+parent: wiki
+mod: ace
+version:
+  major: 3
+  minor: 17
+  patch: 0
+---
+
+## 1. Config Values
+### 1.1 Make item attachable
+
+An item can be added to ACEs "Attach to Vehicle" and "Attach to Self" system by adding the following config value to the ``CfgWeapons`` or ``CfgMagazines`` configs:
+```cpp
+class CfgWeapons {
+    class attach_item: CBA_MiscItem {
+        ACE_attachable = "new_attachable_item_classname";
+    };
+};
+```
+
+The classname in ``ACE_attachable`` has to match a config value in ``CfgVehicle`` which to be able to spawn the physical item in the world. Example:
+```cpp
+class CfgVehicles {
+    class ThingX;
+	// The object that gets created and attached in ACE
+    class new_attachable_item_classname: ThingX {
+        scope = HIDDEN;
+        displayName = "New ACE attachable item";
+        model = QPATHTOF(data\model_file.p3d);
+		vehicleClass = "";
+    };
+};
+```
+
+### 1.2 Define attach orientation for non-symmetric items
+In the case the item needs to have a particular orientation when attached, add the config value: ``ACE_attachable_orientation`` which is an array describing the ``vectorDir`` orientation of the object.  
+The default value is: ``[1,0,0]``. 
+
+Example: 
+```cpp
+class CfgWeapons {
+    class attach_item: CBA_MiscItem {
+        ACE_attachable = "new_attachable_item_classname";
+        ACE_attachable_orientation[] = {-1,0,0};
+    };
+};
+```
+
+## 2. Event Handlers
+### 2.1 Attach Events   
```

Comment:
Not looking good in preview. Self-reminder to check later.

Reply:
The view-file previewer on github doesn't seem to render tables well either. Check the ``events-framework.md`` with view-file and you see the same issue. But I haven't touched that file at all and current wiki shows it fine. 
![billede](https://github.com/acemod/ACE3/assets/7889925/094ced61-726b-4d4d-8382-4f657238bf57)

Reply:
Worst case scenario, we fix it later.

Reply:
`docker-compose up` in the `docs` folder and check. ;)

Reply:
Checked with docker, table looks fine! 
![billede](https://github.com/acemod/ACE3/assets/7889925/6f171c9a-bd8e-40dd-ba58-a9d8abb13f62)